README.md

```diff
@@ -333,4 +343,5 @@ if ($success) {
 }
```

Comment:
afterCommit expects param 3 to be of type ArrayObject

Reply:
$table->behaviors() should be $this->Bookmarks to match the previous line, or not?